src/operator/quantization/quantize.cc

```diff
@@ -47,6 +47,24 @@ bool QuantizeStorageType(const nnvm::NodeAttrs& attrs,
   return true;
 }
 
+#if MXNET_USE_ONEDNN == 1
+static void QuantizeComputeExCPU(const nnvm::NodeAttrs& attrs,
+                                 const OpContext& ctx,
+                                 const std::vector<NDArray>& inputs,
+                                 const std::vector<OpReqType>& req,
+                                 const std::vector<NDArray>& outputs) {
+  const QuantizeParam& param = nnvm::get<QuantizeParam>(attrs.parsed);
+
+  if (SupportDNNLQuantize(param.out_type)) {
```

Comment:
you should call GetQuantizeOutputType(param) here as param.out_type can be set to auto, which won't be supported in this case.  GetQuantizeOutputType convert auto to the proper type

Reply:
The problem is that `QuantizeParam` does not include fields needed for the `GetQuantizeOutputType` (I mean `min_calib_range` and `max_calib_range`) so it would also involve changing `QuantizeParam`

Reply:
Ok, I confused it with QuantizeV2 :)